### ISSUE TITLE
Consider saving github credentials as an option for reuse

### ISSUE THREAD

**Issue by chasevida** [chasevida/orient#5]
After using this a few times on different projects it seems that saving a users github credentials somewhere may make this a lot easier and faster to use. Perhaps adding this as an option would be best.

**Comment by chasevida** [comment]
Not sure this is well suited for the module that it's dependent on. Perhaps worth picking this up later again.

**Comment by yuvilio** [comment]
I agree with this feature request. Want to use the command line but don't like inputing credentials in the command. There are a few libraries like [github-oauth](https://github.com/maxogden/github-oauth) that might fit here.

**Comment by chasevida** [comment]
Thanks @yuvilio for the suggested lib. I do want to add this in because it frustrates me also. It's more a matter of finding a little bit of time to add this in. I will try take a look at it this week or next and post back here when it's done.

**Comment by chasevida** [comment]
Hey @yuvilio, I've had a look into some options here but I honestly have not had any time to implement anything solid with my schedule. I'd be happy to take a look at a PR if you want to give it a shot.
